api/snyk.ts

```diff
@@ -0,0 +1,49 @@
+import cheerio from 'cheerio'
+import got from '../libs/got'
+import { createBadgenHandler, PathArgs } from '../libs/create-badgen-handler'
+
+export default createBadgenHandler({
+  title: 'Snyk',
+  examples: {
+    '/snyk/badgen/badgen.net': 'vulnerability scan',
+    '/snyk/babel/babel/6.x': 'vulnerability scan (branch)',
+    '/snyk/tunnckoCore/opensource/master/@tunnckocore%2Futils%2Fpackage.json': 'vulnerability scan (custom path)'
+  },
+  handlers: {
+    '/snyk/:user/:repo/:branch?/:targetFile?': handler
+  }
+})
+
+async function handler ({ user, repo, branch = 'master', targetFile }: PathArgs) {
+  const badgeUrl = `https://snyk.io/test/github/${user}/${repo}/${branch}/badge.svg`
+  const searchParams = new URLSearchParams()
+  if (targetFile) searchParams.set('targetFile', targetFile)
```

Comment:
Correct me if I'm wrong, but that seems to be supported? :thinking:

Reply:
That is setting query param for an outbound request. But actual value isn't coming from inbound request's query param but `targetFile` path argument instead.